analyzers/src/SonarAnalyzer.CSharp/Rules/SillyBitwiseOperation.cs

```diff
@@ -76,5 +76,10 @@ private void CheckBinary(SyntaxNodeAnalysisContext context, int constValueToLook
             var binary = (BinaryExpressionSyntax)context.Node;
             CheckBinary(context, binary.Left, binary.OperatorToken, binary.Right, constValueToLookFor);
         }
+
+        private static bool IsFieldOrProperty(SemanticModel semanticModel, SyntaxNode node) =>
+            semanticModel.GetSymbolInfo(node).Symbol is { } symbol
+            && symbol.ContainingNamespace.Name != "System"  // Excluding built-in constants
```

Comment:
I don't understand this case. Can you give an example?

Reply:
Sure
`
resultNint = bitMaskNint | IntPtr.Zero;   // Noncompliant`

Reply:
Where  IntPtr belongs to System namespace, but Zero is a field there

```
[Intrinsic]
public static readonly IntPtr Zero;
```